api4/team_test.go

```diff
@@ -1220,18 +1220,74 @@ func TestSearchAllTeamsPaged(t *testing.T) {
 		teams[i] = newTeam
 	}
 
+	foobarTeam, err := th.App.CreateTeam(&model.Team{
+		DisplayName: "FOOBAR",
+		Name:        "whatever",
+		Type:        model.TEAM_OPEN,
+		Email:       th.GenerateTestEmail(),
+	})
+	require.Nil(t, err)
+
 	testCases := []struct {
 		Name               string
 		Search             *model.TeamSearch
 		ExpectedTeams      []string
 		ExpectedTotalCount int64
 	}{
+		{
+			Name:               "Get foobar channel",
```

Comment:
is it "team" instead of "channel"?

Reply:
Ticket made https://mattermost.atlassian.net/browse/MM-26720